### ISSUE TITLE
How to use multiple serial ports in qgroundcontrol under Android

### ISSUE THREAD

**Issue by xzxzhao** [mavlink/qgroundcontrol#12015]
I use two serial ports on my android tablet but qgroundcontrol can only communicate with one of them

**Comment by HTRamsey** [comment]
Yup, currently not possible but I can look into it. Multiple serial ports on Android is more common than it used to be
